sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/DisposableTrainedModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Training;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Tests
+{
+    public class DisposableTrainedModel : IAsyncDisposable
```

Comment:
I'm not sure how we think about access modifiers in tests in the SDK, but should this be internal?

Reply:
I don't think we are that concerned since the test project won't be used by others. I could make it `internal`, but it would be a bit arbitrary. Technically speaking, everything could be `internal`, except for the tests maybe.